src/data/config.js

```diff
@@ -11,6 +11,7 @@ import { ACTIONS } from './constants';
  */
 export const DEFAULTS = {
   panorama           : null,
+  addRequestHeaders  : null,
```

Comment:
I prefer this is added toward the end of the list (most advanced settings)

The type of the parameter should be tested at init in the validators (bellow in this file)

Reply:
No problem I will move it down